teps/0152-tekton-conformance-verification.md

```diff
@@ -0,0 +1,166 @@
+---
+status: implementable
+title: Tekton Conformance Verification
+creation-date: '2024-02-12'
+last-updated: '2024-02-12'
+authors:
+- '@JeromeJu'
+---
+
+# TEP-0152: Tekton Conformance Verification
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Verification Testing Strategy](#verification-testing-strategy)
+    - [Alternatives Considered](#alternatives-considered)
+  - [Vendor Requirements](#vendor-requirements)
+  - [Conformance verification Test coverage](#conformance-verification-test-coverage)
+  - [Test Sutie Design Details](#test-sutie-design-details)
+  - [Tekton Conformance Verification Versioned](#tekton-conformance-verification-versioned)
+- [Future Work](#future-work)
+
+## Summary
+
+This TEP outlines the Tekton Conformance verification process according to [Tekton v1 conformance API spec](https://github.com/tektoncd/pipeline/blob/main/docs/api-spec.md). It details the implementation of the Tekton conformance verification test suite, paving the way for a community-driven Tekton conformance certification process.
+
+## Motivation
+
+With the release of [v1 Tekton conformance API spec](https://github.com/tektoncd/pipeline/blob/main/docs/api-spec.md), the Tekton community can now offer Tekton conformance verification. The conformance test suite will be the crucial prerequisite for vendors to claim conformance for their service.
+
+### Goals
+- Defines a **detailed conformance verification process** that Tekton vendors can verify and thereby claiming Tekton conformance for their service.
+- Establishes a clear versioning policy for the Conformance Policy, including its verification process, to streamline future updates.
+
+## Proposal
+
+### Verification Testing Strategy
```

Comment:
I am not sure I follow what the `ProcessAndSendToTekton` does ? 🤔

Reply:
It leaves to the vendors to implement their wrapping of all the service detail, intaking and outputing Tekton YAML for verifying conformance fields.

I can add more abstract summary to the TEP.

🙏  thanks for the question